README.md

```diff
@@ -104,7 +104,7 @@ The following limitations will be addressed in future releases (see [Roadmap](#r
 - [x] Dockerfile
 - [x] github actions
 - [x] metrics
-- [ ] support multiple matchers
+- [x] support multiple matchers
```

Comment:
Could you please also remove the following line from README.md?

> At the moment thanos-federate-proxy does not support multiple matcher queries and will only apply the first one.

Reply:
This is what these changes enable and that's why it's added to the readme.

Reply:
ah, ok, I didn't notice that extra sentence, removed.